server/services/store/storetests/blocks.go

```diff
@@ -310,3 +317,128 @@ func testGetRootID(t *testing.T, store store.Store, tearDown func()) {
 		require.Error(t, err)
 	})
 }
+
+func testGetParentID(t *testing.T, store store.Store) {
+	userID := "user-id"
+
+	blocks, err := store.GetAllBlocks()
+	require.NoError(t, err)
+	initialCount := len(blocks)
+
+	blocksToInsert := []model.Block{
+		{
+			ID:         "parent",
+			RootID:     "parent",
+			ModifiedBy: userID,
+		},
+		{
+			ID:         "child1",
+			RootID:     "parent",
+			ParentID:   "parent",
+			ModifiedBy: userID,
+		},
+		{
+			ID:         "child2",
+			RootID:     "parent",
+			ParentID:   "parent",
+			ModifiedBy: userID,
+		},
+		{
+			ID:         "grandchild1",
+			RootID:     "parent",
+			ParentID:   "child1",
+			ModifiedBy: userID,
+		},
+		{
+			ID:         "grandchild2",
+			RootID:     "parent",
+			ParentID:   "child2",
+			ModifiedBy: userID,
+		},
+		{
+			ID:         "greatgrandchild1",
+			RootID:     "parent",
+			ParentID:   "grandchild1",
+			ModifiedBy: userID,
+		},
+	}
+
+	InsertBlocks(t, store, blocksToInsert)
+
+	blocks, err = store.GetAllBlocks()
+	require.NoError(t, err)
+	require.Len(t, blocks, initialCount+6)
+
+	t.Run("from root id", func(t *testing.T) {
+		parentID, err := store.GetParentID("parent")
+		require.NoError(t, err)
+		require.Equal(t, "", parentID)
```

Comment:
@chenilim is this the expected behavior?

Reply:
Yes, the parentID is null (returned as "") for root blocks.